tox.ini

```diff
@@ -11,7 +11,7 @@ deps =
   mkdocs-git-revision-date-localized-plugin
   mkdocs-glightbox
   mkdocs-macros-plugin
-  mkdocs-material~=9.4.0
+  git+https://github.com/squidfunk/mkdocs-material@master
```

Comment:
@colder-is-better: Just a heads-up that this exists. I will update this line to `mkdocs-material>=9.5.3` once that version is released, and then I will mark this PR ready for review.

Reply:
Ack, thanks.